src/primitives.jl

```diff
@@ -107,7 +107,8 @@ the new struct but are useful to give context to plots.
 Called at the array level the returned tuple will also include the
 previous reference dims attached to the array.
 """
-# Results are split as (dims, refdims)
+function slicedims(A, I) end
```

Comment:
Because of this definition, if calling `sliceddims` with in correct arguments, it will not throw a method error as it should but instead return `nothing`. If you want a function definition only for the documentation string, it is better practice to do 
```julia
function slicedims end
```

instead.